bootstrap/sql/com.mysql.cj.jdbc.Driver/v009__create_db_connection_info.sql

```diff
@@ -229,4 +229,6 @@ UPDATE ml_model_entity SET json = REGEXP_REPLACE(json, "\"source\"\\s*:\\s*\"Tag
 -- Delete supportsProfiler from Mssql
 UPDATE dbservice_entity
 SET json = JSON_REMOVE(json, '$.connection.config.uriString')
-WHERE serviceType = 'Mssql';
\ No newline at end of file
+WHERE serviceType = 'Mssql';
```

Comment:
@mohityadav766 are these new, if so they should be in v009 right , anyone from 1.0.0 won't get this migration right

Reply:
moved migrations for remove subscriptions to new file